arex-instrumentation/dubbo/arex-dubbo-common/src/main/java/io/arex/inst/dubbo/common/AbstractAdapter.java

```diff
@@ -118,7 +118,31 @@ public Map<String, String> getRequestHeaders() {
     }
 
     public String getPath() {
-        return StringUtil.defaultIfEmpty(getAttachment("path"), getServiceName());
+        return StringUtil.defaultIfEmpty(this.getServiceNameWithVersion(), this.getServiceName());
+    }
+
+    /**
+     * <p>Illustrate: Splicing serviceName and version
+     * <p>Fix Bug: https://github.com/arextest/arex-agent-java/issues/490
+     * <p>Notice: This method maybe affect ignore [path] config！ Especially when a dubbo service exists multiple versions！
+     */
+    protected String getServiceNameWithVersion(){
```

Comment:
I think this method can directly use `this.getServiceName()`, there is no need to parse it again?